datahub-web-react/src/app/permissions/roles/ManageRoles.tsx

```diff
@@ -21,7 +21,9 @@ import { OnboardingTour } from '../../onboarding/OnboardingTour';
 import { ROLES_INTRO_ID } from '../../onboarding/config/RolesOnboardingConfig';
 import { clearUserListCache } from '../../identity/user/cacheUtils';
 
-const SourceContainer = styled.div``;
+const SourceContainer = styled.div`
+    height: calc(100% - 40px);
```

Comment:
where is 40px coming from?

Reply:
make PageContainer height: calc(100% - 11vh);  SourceContainer height: 100%